src/main/java/gregtech/integration/jei/multiblock/infos/LargeMinerInfo.java

```diff
@@ -0,0 +1,59 @@
+
+package gregtech.integration.jei.multiblock.infos;
+
+import com.google.common.collect.Lists;
+import gregtech.api.metatileentity.multiblock.MultiblockControllerBase;
+import gregtech.common.blocks.MetaBlocks;
+import gregtech.common.metatileentities.MetaTileEntities;
+import gregtech.common.metatileentities.multi.electric.MetaTileEntityLargeMiner;
+import gregtech.integration.jei.multiblock.MultiblockInfoPage;
+import gregtech.integration.jei.multiblock.MultiblockShapeInfo;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.EnumFacing;
+
+import java.util.List;
+
+
+public class LargeMinerInfo extends MultiblockInfoPage {
+
+    private final MetaTileEntityLargeMiner largeMiner;
+
+    public LargeMinerInfo(MetaTileEntityLargeMiner largeMiner) {
+        this.largeMiner = largeMiner;
+    }
+
+    @Override
+    public MultiblockControllerBase getController() {
+        return largeMiner;
+    }
+
+    @Override
+    public List<MultiblockShapeInfo> getMatchingShapes() {
+        MultiblockShapeInfo shapeInfo = MultiblockShapeInfo.builder()
+                .aisle("EPO", "#F#", "#F#", "#F#", "###", "###", "###")
+                .aisle("SPP", "FPF", "FPF", "FPF", "#F#", "#F#", "#F#")
+                .aisle("IPP", "#F#", "#F#", "#F#", "###", "###", "###")
```

Comment:
The shape info needs to be changed into the new XYZ system

Apart from this, this looks ready to merge from my testing